tasks-dev/assemble.js

```diff
@@ -53,7 +52,6 @@ gulp.task('assemble', () => {
         //     ext: '.html'
         // }))
 
-        .pipe(debug())
```

Comment:
Does removing this mean that the task won’t now output what files are being changed?